doc/python/creating-and-updating-figures.md

```diff
@@ -487,34 +559,48 @@ fig.for_each_trace(
 fig.show()
 ```
 
-#### The update axis methods
+#### Updating Figure Axes
 
-Graph object figures support `update_xaxes` and `update_yaxes` methods that may be used to update multiple nested properties of one or more of a figure's axes. Here is an example of using `update_xaxes` to disable the vertical grid lines across all subplots in a figure produced by Plotly Express.
+Graph object figures support `update_xaxes()` and `update_yaxes()` methods that may be used to update multiple nested properties of one or more of a figure's axes. Here is an example of using `update_xaxes()` to disable the vertical grid lines across all subplots in a figure produced by Plotly Express.
 
 ```python
 import pandas as pd
 import plotly.express as px
+
 df = px.data.iris()
+
 fig = px.scatter(df, x="sepal_width", y="sepal_length", color="species", facet_col="species")
+
 fig.update_xaxes(showgrid=False)
+
 fig.show()
 ```
 
-There are also `for_each_xaxis` and `for_each_yaxis` methods that are analogous to the `for_each_trace` method described above. For non-cartesian subplot types (e.g. polar), there are additional `update_{type}` and `for_each_{type}` methods (e.g. `update_polar`, `for_each_polar`).
+There are also `for_each_xaxis()` and `for_each_yaxis()` methods that are analogous to the `for_each_trace()` method described above. For non-cartesian subplot types (e.g. polar), there are additional `update_{type}` and `for_each_{type}` methods (e.g. `update_polar()`, `for_each_polar()`).
 
-### Other update methods
+### Other Update Methods
 
-`go` figures also support `update_layout_images` in order to [update background layout images](/python/images/), `update_annotations` in order to [update annotations](/python/text-and-annotations/#multiple-annotations), and `update-shapes` in order to [update shapes](/python/shapes/).
+Figures created with the `plotly.graph_objects` module also support:
```

Comment:
This isn't totally correct: all figures support this, including those created with make_subplots and PX...